Clients/Xamarin.Interactive.Client.Web/ClientApp/WorkbookSession.ts

```diff
@@ -124,6 +144,78 @@ export class WorkbookSession {
         await this.hubConnection.invoke('InitializeSession', description)
     }
 
+    initWasm(): void {
+        if (!this.wasmFrame) {
+            this.wasmFrame = document.createElement("iframe")
+            this.wasmFrame.sandbox.add("allow-scripts", "allow-same-origin")
+            this.wasmFrame.setAttribute("seamless", "true")
+            this.wasmFrame.src = "/Home/Wasm"
+            this.wasmFrame.style.display = "none";
+            this.wasmFrame.addEventListener("load", this.executeQueuedWasmMessages)
+            this.wasmMessageChannel = new MessageChannel
+            this.wasmMessageChannel.port1.onmessage = this.dispatchWasmMessage
+            document.body.appendChild(this.wasmFrame)
+        }
+    }
+
+    executeQueuedWasmMessages(ev: any): void {
+        this.wasmFrameLoaded = true
+        this.statusUIActionEvent.dispatch({
+            action: StatusUIAction.DisplayMessage,
+            message: {
+                kind: MessageKind.Status,
+                severity: MessageSeverity.Info,
+                text: "Initializing WebAssembly agent…",
+                showSpinner: true
+            }
+        });
+        this.wasmFrame!.contentWindow!.postMessage({ $type: "ChannelOpenMessage" }, location.origin, [
```

Comment:
We need to transfer the other port of the message channel to the WASM frame, so we need this one `postMessage` call—everything afterwards goes through the message channel, which should be more efficient, and doesn't mess with Draft/Monaco.